Makefile

```diff
@@ -7,7 +7,8 @@ PLATFORM = linux
 PLATFORM_DIR = platform_$(PLATFORM)
 
 help::
-	@echo 'Usage: make [debug]'
+	@echo 'Usage: make [<target>]'
+	@echo 'Supported targets: clean all libs all-tests run-tests test-results install'
```

Comment:
Hard-coded for now. There are `Makefile` scripts / logic that can generate this programatically, but I didn't want to go that far in this round. (Less imp, may add that in future.)